tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
+
+\noindent\textbf{Reward:} 750 points (250 points per guest).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Robot providing unique description of a person without having asked such information (75 points)
```

Comment:
This reward is way too small for the complexity. Is easier to find a person than to describe her.

Consider the following:
- 50pts navigating back to the operator and provide requested information.
- 50pts for retrieving the list of guests (automatically scored if good ASR).
- 150pts for providing a a unique and comprehensive description of a guest (e.g. gender, clothing, skin, or hair color, pose, etc).

Reply:
Aren't we providing points just for the Main goal achieved? This sounds to me like going back to scoring lots of little things.

Reply:
For consistency with other tests, we want a test to be
- Main goal: ~500pts
- Bonus goals: ~500pts
- Maximum total score: **exactly 1000pts**

Main goal, as I understand the test, is to find a person. Additionally, the robot may get the person's description.
- If the robot fails to find a person, may ask the person to wave (score reduction)
- If the robot fails to find a person, may ask the person to stand in front (no points)
- Whether the robot found the person or not, the robot may ask the person for a description
- If the robot is fast enough, it can navigate back and deliver the person's location and description for points

Otherwise you'll have teams frustrated because the robot failed to either find the person, or get a description of her. If, on top, you look in the entire apartment, is too much for a Stage 1 test.

Reply:
A simple and clear rewarding scheme can be like, 

__Rewards__ _500 points (100 pts for location and 150 pts for description per guest)._

__bonus rewards__ _For retrieving information about each additional guest there will be 100 points for location and 150 pts for description. A total of 500 bonus pts._

However, if we drop the reporting of location then will it be justified to give away 250 pts just for the description?

Reply:
I rather focus in giving points based on how hard is to accomplish a task, then round to sum.

Yet again I think that a reward of 100 points for finding a person in the whole apartment is scarce. I'd say the same for a semantically meaningful description of the people (i.e. listing descriptors is **not** a valid approach).

Reply:
100 points are for finding the guests in the same room.

Reply:
Assuming that finding the third person will be very challenging in 5 mins, we can give bonus of 300 pts for description and 200 pts for location for reporting the third guest.